src/renderer/src/pages/game-details/hero-panel-actions.tsx

```diff
@@ -47,7 +47,7 @@ export function HeroPanelActions({
         filters: [
           {
             name: "Game executable",
-            extensions: window.electron.platform === "win32" ? ["exe"] : [],
+            extensions: ["exe"],
```

Comment:
Not sure about this one. Not all of the game binaries are `.exe` on Windows

Reply:
Do you have any examples? We could have no filter but i think it makes the experience annoying.

Reply:
Not really to be honest, but I think that it's safer to left it open wide on anything else than Windows